zenoh/src/lib.rs

```diff
@@ -403,6 +410,7 @@ impl InitBuilder {
 
     /// Restrict the matching queries that will be receive by the [`Queryables`](crate::queryable::Queryable)
     /// to the ones that have the given [`Locality`](crate::prelude::Locality).
+    #[cfg(feature = "unstable")]
```

Comment:
Locality should not be configurable at session level but only per declare_subscriber and declare_queryable-basis.